edpop-record-ontology.ttl

```diff
@@ -0,0 +1,140 @@
+# somehow normalized, human-readable representation of the property, but they are not
+# thesaurized and certainly not guaranteed to be the same if they are conceptually the same
+# (e.g., Köln or Keulen for the city of Cologne).
+
+# My proposal would be to use these properties that can be extracted from the original record
+# in a deterministic way. That means: only string literals, and also links to other records
+# if they are defined in the original catalogues (such as links to CT Thesaurus).
+# Users will be able to enrich, correct and further normalize this data with annotations.
+# Annotations may also be used, perhaps, for automatic non-deterministic normalizations,
+# such as the place of publication as a link to Wikidata.
+
+edpoprec:title a rdf:property ;
+               skos:prefLabel "Title"@en ;
+               rdfs:domain edpoprec:BibliographicalRecord ;
+               rdfs:range xsd:string .
+
+edpoprec:alternativeTitle a rdf:property ;
+                            skos:prefLabel "Alternative title"@en ;
+                            rdfs:domain edpoprec:BibliographicalRecord ;
+                            rdfs:range xsd:string .
+
+# Or: contributor? And how to define the role of the contributor? Perhaps a separate
+# class is better. The catalogues generally describe the roles of the contributors quite
+# well (author, translator, illustrator, etc.)
+edpoprec:author a rdf:property ;
+                skos:prefLabel "Author"@en ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
+
+edpoprec:publisherOrPrinter a rdf:property ;
+                            skos:prefLabel "Publisher or printer"@en ;
+			    rdfs:domain edpoprec:BibliographicalRecord ;
+			    rdfs:range xsd:string .
+
+edpoprec:placeOfPublication a rdf:property ;
+                            skos:prefLabel "Place of publication"@en ;
+			    rdfs:domain edpoprec:BibliographicalRecord ;
+			    rdfs:range xsd:string .
+
+# Dating has xsd:string as its range, because the date is very often not a simple year
+# in the catalogues but can be anything, like [1650] for an estimation or a range like
+# 1800-1914.
+edpoprec:dating a rdf:property ;
+                skos:prefLabel "Dating"@en ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
+
+# For language, not sure which class to use. There are various IRI collections, of which
+# ISO 639-3 (e.g. https://iso639-3.sil.org/code/nld) and Glottolog
+# (e.g. https://glottolog.org/resource/languoid/id/mode1257) seem most attractive,
+# or perhaps Wikidata. Or just the ISO 639-3 code as a literal?
+edpoprec:language a rdf:property ;
+                  skos:prefLabel "Language"@en ;
+		  rdfs:domain edpoprec:BibliographicalRecord .
+
+edpoprec:format a rdf:property ;
+                skos:prefLabel "Format"@en ;
+		skos:description "Format or physical description" ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
```

Comment:
Format like in size? Format like in form? Something else?

Reply:
That's not really clear indeed. I had thought of it as a general description of the physical object. But I see that most catalogues (at least all that use Marc21) split this out in (1) extent (number of pages); (2) size (generally the longest side in centimetres) and (3) additional remarks, such as that a resource has illustrations. All three are essential for our purposes. So I think we could as well make three properties for this.